.github/workflows/mac-tests.yml

```diff
@@ -46,6 +46,11 @@ jobs:
         optuna --version
         pip install --progress-bar off .[tests]
         pip install --progress-bar off .[optional]
+
+        # TODO(nzw0301) remove the constraint of sklearn `1.0.0`
+        # after skopt supports sklearn >= 1.0.0
+        pip install "scikit-learn<1.0.0"
```

Comment:
Can or should we not update the `optional` extras requirements in `setup.py` instead?

Reply:
Definitely. I've updated it. Thanks!